low-level/solution/lr2/server/health.py

```diff
@@ -680,23 +680,31 @@ def get_disks_info(self):
             disk_health = self.get_health_template(uid, True)
             health_data = disk.get_health()
             health = "OK" if (health_data['SMART_health'] == "PASSED") else "Fault"
-            self.set_health_data(disk_health, health, specifics=[{"SMART": health_data}])
+            serial_no = disk.id.split("-")[-1] if disk.id else "NA"
```

Comment:
check if required key (like  serial_no here) exists and if not, then only over ride, as a general practice for all resources where this sorting logic would be used

Reply:
Done